datacapturing/src/androidTestMovebis/java/de/cyface/datacapturing/MovebisTest.java

```diff
@@ -107,6 +107,7 @@ public void run() {
      * @throws SetupException Should not happen. For further details look at the documentation of
      *             {@link MovebisDataCapturingService#MovebisDataCapturingService(Context, String, UIListener, long)}.
      */
+    @Test
```

Comment:
Ohne das @Test bekomme ich einen Fehler beim Ausführen der iTs dass der Test keine Testcases enthält. Ist vermutlich unabsichtlich rausgeflogen das @Test? in Bitrise habe ich einen Gradle Filter der alle Flaky Tests getrennt ausführt und deren success ignoriert.